test/vtcomposite-localize-worldview.test.js

```diff
@@ -0,0 +1,352 @@
+'use strict';
+
+const localize = require('../lib/index.js').localize;
+const { vtinfo, getFeatureById } = require('./test-utils.js');
+const mvtFixtures = require('@mapbox/mvt-fixtures');
+const test = require('tape');
+
+const fixtureDefaults = mvtFixtures.create({
```

Comment:
i don't think I knew you could do this. neato!

Reply:
Added in https://github.com/mapbox/mvt-fixtures/pull/105!